### PR TITLE
Add extra dependencies

### PR DESCRIPTION
**Pull request by daniviga**
See #3

### CODE REVIEW

File: helpers/whldownload.sh [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+        *)
+            shift;;
+    esac
+done
+
+checkcmd curl
+
+for d in "${WH[@]}"; do
+    cd $d
+    while read l
```
Comment (nastasi-oq) [inline helpers/whldownload.sh]: please rearrange with the list file and "; do" on the same line (using cat <filename> | while ... ).

File: helpers/whlsetup.sh [hunk 2]
```diff
@@ -1,13 +1,32 @@
...
+# You should have received a copy of the GNU Affero General Public License
+# along with OpenQuake. If not, see <http://www.gnu.org/licenses/>.
 
-set -x
+if [ $GEM_SET_DEBUG ]; then
+    set -x
+fi
 set -e
 
 help() {
```
Comment (nastasi-oq) [inline helpers/whlsetup.sh]: Help does not work as previews script, already: I suggest to accept a parameter that will be returned with exit.

File: helpers/whlsetup.sh [hunk 3]
```diff
@@ -21,10 +40,16 @@ checkpath() {
...
+}
+
 if [ $# -lt 5 -o "$1" == "-h" -o "$1" == "--help" ]; then
-    help;
+    help
 fi
 
+declare -a SRC
+
 while (( "$#" )); do
```
Comment (nastasi-oq) [inline helpers/whlsetup.sh]: while [ $# -gt 0 ]; do

File: helpers/whlsetup.sh (hunk 2 above)
Comment (daniviga) [inline helpers/whlsetup.sh]: Done

**Pull request merged**